types/compilation/compilation.interfaces.ts

```diff
@@ -231,7 +231,7 @@ export type BuildResult = CompilationResult & {
     downloads: BuildEnvDownloadInfo[];
     executableFilename: string;
     compilationOptions: string[];
-    stdout: ResultLine[];
+    stdout?: ResultLine[];
     stderr: ResultLine[];
```

Comment:
It's odd to me that stdout might be undefined while stderr is not. I am wondering if there is a mistake elsewhere allowing an undefined value to reach here when the intent was different? I'd need to look a little more at the code here.